frontend/gateway/pb/gateway.proto

```diff
@@ -31,6 +36,8 @@ message SolveRequest {
 	string Frontend = 2;
 	map<string, string> FrontendOpt = 3;
 	repeated string ImportCacheRefs = 4;
+	
+	// apicaps.CapSolveInlineReturn deprecated
```

Comment:
`apicaps:CapSolveInlineReturn`? (Are these `apicaps:` comments expected to be machine-readable?)

Reply:
No, the comments are just to visually track what part of the API corresponds to the cap. `CapSolveInlineReturn` is meant for this comment. To implement that I plan to add a `Return`  method and the current behavior of returning with `final` parameter inside the solve method would be covered by `CapSolveInlineReturn`.